docs/ref/templates/builtins.txt

```diff
@@ -212,6 +212,22 @@ This tag can be used in two ways:
 
 See :ref:`template-inheritance` for more information.
 
+Literal form (with quotes) can use relative path (started from ``./`` or ``../``).
+
+Extend "base.html":
+
+* ``{% extends "./base.html" %}`` located in the same folder,
+  where your template placed
+
+* ``{% extends "../../base.html" %}`` located at two levels higher
```

Comment:
English wash:

* ``{% extends "../../base.html" %}``: "base.html" is located at two levels higher